### DISCUSSION TITLE
Running this on a cloud mac

### DISCUSSION THREAD

**Discussion by clouedoc** [godbout/AccessibilityStrategyTestApp#73] [General]
Hey there,

I've seen this line inside your README.md:

> the AccessibilityStrategy now requires macOS 12 minimum. GitHub Actions only goes up to 11 (haha). so back to running locally for now.
>>GUILO

Did you know that you could rent a cloud M1 (e.g. on Scaleway) and connect it to GitHub as a GitHub action runner?
That might make your life easier.

Anyway, I'm off to viming. Have a good day 😄

**Comment by godbout** [comment]
yeah tbh i didn't really bother when i switched to Monterey. i was paying already for GitHub, but i could still run the tests locally. but once i switch to Ventura i'm gonna need a CI again yeah, so that i can test on both platforms. Xcode has some integrated stuff, Xcode Cloud. not sure about the costs tho. they all seem to be costing a fortune.
